src/conn/conn_api.c

```diff
@@ -1982,12 +1982,52 @@ __wt_debug_mode_config(WT_SESSION_IMPL *session, const char *cfg[])
     return (0);
 }
 
+/*
+ * __wt_json_config --
+ *     Set JSON output configuration.
+ */
+int
+__wt_json_config(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
+{
+    static const WT_NAME_FLAG jsontypes[] = {
+      {"error", WT_JSON_OUTPUT_ERROR}, {"message", WT_JSON_OUTPUT_MESSAGE}, {NULL, 0}};
+
+    WT_CONFIG_ITEM cval, sval;
+    WT_CONNECTION_IMPL *conn;
+    WT_DECL_RET;
+    const WT_NAME_FLAG *ft;
+    uint8_t flags;
+
+    conn = S2C(session);
+
+    /*
+     * When reconfiguring, check if there are any configurations we care about, otherwise leave the
+     * current settings in place.
+     */
+    ret = 0;
```

Comment:
It's good practice but `WT_DECL_RET` already inits the value to 0.

Reply:
Thanks -- nice catch, now that the code is in a separate function that separate initialization is no longer needed. Fixed in aaa1b49ae.